src/theme/DocSidebar/index.js

```diff
@@ -8,7 +8,7 @@ export default function DocSidebarWrapper(props) {
       <div className="sidebar-recommendation-label">
         ✨ Denotes recommended application <br />
         ♊ Denotes availability on Gemini-3h testnet <br />
-        ♉ Denotes availability on Taurus testnet
+        ♉ Denotes availability on Taurus testnet / Mainnet
```

Comment:
No, this won't be the case, mainnet will use another emoji

Reply:
I figured as much, but we wanted to get this PR out as soon as possible. It also serves as the updates for today's mainnet release.